docs/src/_guide/configuration.md

```diff
@@ -113,6 +126,7 @@ Environment levels within `meltano.yml` resolve in order of precedence (within a
 - environment level env
 - root level plugin env
 - root level env
+- schedule level env
 - .env file
 - terminal env # lowest
 ```
```

Comment:
@cjohnhanson It won't let me comment on the line I want, but will you add this after the last sentence:

```html
        <div class="notification is-info">
          <p>We are considering adding support for the <code>env</code> key to <a href="/concepts/project#jobs">jobs</a> as well as updating the precedence order when we do. We'd love to hear your thoughts in the <a href="https://github.com/meltano/meltano/issues/6386">GitHub issue</a> about this possible change!</p>
        </div>
```

Reply:
👍 Good call, added.